applications/nrf5340_audio/src/modules/audio_i2s.c

```diff
@@ -77,19 +87,12 @@ static nrfx_i2s_config_t cfg = {
 	.mode = NRF_I2S_MODE_MASTER,
 	.format = NRF_I2S_FORMAT_I2S,
 	.alignment = NRF_I2S_ALIGN_LEFT,
+	.ratio = CONFIG_AUDIO_RATIO,
+	.mck_setup = 0x66666000,
 #if (CONFIG_AUDIO_BIT_DEPTH_16)
 	.sample_width = NRF_I2S_SWIDTH_16BIT,
-	.mck_setup = 0x66666000,
-	.ratio = NRF_I2S_RATIO_128X,
-#elif (CONFIG_AUDIO_BIT_DEPTH_24)
```

Comment:
So we are removing support for bit depth of 24?
If so, maybe remove it from audio/Kconfig?

Reply:
Yes, it will mess up the ratios when changing sample rate and it is not a good implementation with the current clock mismatch

Reply:
Did a search and removed all instances of AUDIO_BIT_DEPTH_24

Reply:
Fixed